### ISSUE TITLE
_restore_backup() with Postgres connector doesn't read propper db name

### ISSUE THREAD

**Issue by adi-** [django-dbbackup/django-dbbackup#259]
Hi,

I have two databases with Django's routing enabled. My settings are as follows:

```
DATABASES = {
    'default': {
        'ENGINE': 'django.db.backends.postgresql_psycopg2',
        'NAME': 'db_default',
        'USER': ...

    },
    'events': {
        'ENGINE': 'django.db.backends.postgresql_psycopg2',
        'NAME': 'db_events',
        'USER': ...
    }
}
```

What I have found out is `_restore_backup()` function doesn't properly create restore command. If I want to restore `events` db I use `call_command("dbrestore", uncompress=True, servername="www-prd", database="events", interactive=False)` but django-dbbackup produces command: `psql db_default --host=localhost --port=5432 --username=... --no-password --set ON_ERROR_STOP=on --single-transaction`.

I guess there is a problem with `self.settings['NAME']` in https://github.com/django-dbbackup/django-dbbackup/blob/master/dbbackup/db/postgresql.py

**Comment by adi-** [comment]
The problem is in v3.1.3. It was working on v3.0.3

**Comment by ZuluPro** [comment]
Hello @adi- ,
Thanks for this bug reporting.

Could you show me, how you do your backup ?

**Comment by adi-** [comment]
`call_command("dbbackup", compress=True, clean=True, servername="www-prd", database="events")`

**Comment by adi-** [comment]
I confirm it is working in `v3.0.3`

**Comment by adi-** [comment]
Any news with that?

**Comment by adi-** [comment]
I see that it was tagged "bug", so I guess you have confirmed my issue?

**Comment by adi-** [comment]
I have made a quick fix that works for me: https://github.com/django-dbbackup/django-dbbackup/pull/260

**Comment by adi-** [comment]
Of course this fix works only o v3.1.3.
